cluster/libvirt-coreos/skydns-rc.yaml

```diff
@@ -1,37 +1,119 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  labels:
-    k8s-app: skydns
-  name: skydns
+  name: kube-dns-v10
```

Comment:
why not bump to v11 and clone in https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/skydns-rc.yaml.in?